api/v3/PaymentProcessor.php

```diff
@@ -126,6 +126,7 @@ function _civicrm_api3_payment_processor_getlist_defaults(&$request) {
  * @throws \API_Exception
  */
 function civicrm_api3_payment_processor_pay($params) {
+  civicrm_api3_verify_mandatory($params, NULL, ['payment_processor_id', 'amount', 'contribution_id']);
```

Comment:
We don't this call if they are all defined as required in _spec - which you have done below

Reply:
Ok, removed.